src/css/App.css

```diff
@@ -708,7 +761,85 @@ a {
   left:0;
   bottom:0;
 }
-.read-more-button:hover:before,button:hover:after{
+/* .read-more-button:hover:before,button:hover:after{
   width:100%;
   transition:800ms ease all;
+} */
+
+.btn-two{
+  text-align: left;
+  font-size: 16px;
+  background: #792525;
+  border: none;
+  color: #fcc810;
+}
+.btn-two:hover{
+border:1px solid #000000;
+background: transparent;
+color: #000000;
+}
+.btn-three{
+  background: transparent;
+  border: 1px solid#fcc810;
+  color: #000000;
+}
+.btn-three:hover{
+border:1px solid #000000;
+background: transparent;
+color: #000000;
+}
+.btn-four{
+  background: transparent;
+  border: 1px solid grey;
+  color: #000000;
+}
+.btn-four:hover{
+border:1px solid #000000;
+background: transparent;
+color: #000000;
+}
+
+.btn-five{
+  background: gray;
+  border: none;
+}
+.btn-five:hover{
+ background: gray;
+ border: none;
+}
+
+.comment-author{
+  display:inline-block;
+  border-radius: 5px;
+}
+
+.comment-time{
+  color: grey; 
+  font-size: 14px;
+}
+
+.commentor-image{
+  display:inline-block;
+  border-radius: 10px;
+}
+
+.reply-btn{
+ margin-bottom: 3px;
+}
+
+.max-row{
+  max-width: 700px;
+}
+
+.single-article img{
+  width: 62%;
+  margin: 15px 20%;
```

Comment:
@Etomovich, why is pixels and percentages on one attribute?

Reply:
This was the fix Rachel suggested to fix the images of an article problem

Reply:
@Etomovich This was an oversight on my part. Set it the pixels to a percentage say 2%, and see how this works out.